vsts/vsts.yaml

```diff
@@ -341,11 +455,10 @@ jobs:
           inputs:
             script: 'choco install dotnet4.5.1'
 
-      - script: |
-          call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\VsDevCmd.bat"
-          sn.exe -Vr *,31bf3856ad364e35
-
-        displayName: "Disable strong name validation"
+#      - script: |
```

Comment:
Do not need to disable strong name anymore

Reply:
Why remark out instead of remove?

Reply:
updated in latest commit